lib/spack/spack/util/compression.py

```diff
@@ -119,30 +115,23 @@ def _gunzip(archive_file):
 
 
 def _unzip(archive_file):
-    """Try to use Python's zipfile, but extract in the current working
-    directory instead of in-place.
-
-    If unavailable, search for 'unzip' executable on system and use instead
+    """
+    Extract Zipfile, searching for unzip system executable
+    If unavailable, search for 'tar' executable on system and use instead
 
     Args:
         archive_file (str): absolute path of the file to be decompressed
     """
 
     destination_abspath = os.getcwd()
-    try:
-        from zipfile import ZipFile
-        zf = ZipFile(archive_file, 'r')
-        zf.extractall(destination_abspath)
-        zf.close()
-    except ImportError:
-        exe = 'unzip'
-        arg = '-q'
-        if is_windows:
-            exe = 'tar'
-            arg = '-xf'
-        unzip = which(exe, required=True)
-        unzip.add_default_arg(arg)
-        unzip(archive_file)
+    exe = 'unzip'
```

Comment:
Why do we even use unzip?

Reply:
A couple points:

* Some .zip archives are distributed on Linux (so that would have made sense before this PR, since AFAIK tar doesn't handle them)
* .zip files are common on Windows and zip is the most-likely compression utility that is available there (so this also makes sense for this PR)